polkadot/node/core/approval-voting/src/lib.rs

```diff
@@ -313,6 +364,31 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
+			coalesced_approvals_buckets: prometheus::register(
+				prometheus::Histogram::with_opts(
+					prometheus::HistogramOpts::new(
+						"polkadot_parachain_approvals_coalesced_approvals_buckets",
+						"Number of coalesced approvals.",
+					).buckets(vec![1.5, 2.5, 3.5, 4.5, 5.5, 6.5, 7.5, 8.5, 9.5]),
+				)?,
+				registry,
+			)?,
+			coalesced_approvals_waiting_times: prometheus::register(
+				prometheus::Histogram::with_opts(
+					prometheus::HistogramOpts::new(
+						"polkadot_parachain_approvals_coalesced_approvals_waiting_times",
+						"Number of ticks we delay the sending of a candidate approval",
+					).buckets(vec![1.1, 2.1, 3.1, 4.1, 6.1, 8.1, 12.1, 20.1, 32.1]),
+				)?,
+				registry,
+			)?,
+			approved_by_one_third: prometheus::register(
+				prometheus::Counter::new(
+					"polkadot_parachain_approved_by_one_third",
+					"Number of candidates where more than one third had to vote ",
```

Comment:
Why do we need this metric ? I could for example see the benefit of counting the candidates where actual approvals > `needed_approvals`.

Reply:
it was useful for me for debugging, I can drop it if you think we don't need it.